plugins/kubernetes_container.yaml

```diff
@@ -101,5 +107,5 @@ pipeline:
           to: "$labels.stream"
       - move:
           from: log
-          to: "$record"
+          to: "$record.message"
```

Comment:
Does this make a difference? I think $record is fine.

Reply:
It shouldn't, but the ticket specified message.